Задача 4. Есть ли статистически значимые различия в росте
дочерей?
Рост матерей 172, 177, 158, 170, 178,175, 164, 160, 169, 165
Рост взрослых дочерей: 173, 175, 162, 174, 175, 168, 155, 170, 160, 163

In [6]:
from scipy import stats  
import numpy as np
li1 = np.array([172, 177, 158, 170, 178,175, 164, 160, 169, 165])
li2 = np.array([173, 175, 162, 174, 175, 168, 155, 170, 160, 163])

stats.ttest_rel(li1, li2)[1]

0.5228168632983574

Нет статистически значимых различий в росте дочерей 

alpha = 0.05 < p_value = 0.52